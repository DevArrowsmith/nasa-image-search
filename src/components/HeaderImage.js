import styled from 'styled-components';

const HeaderImage = styled.img`
  height: 30px;
  width: auto;
`

export default HeaderImage;