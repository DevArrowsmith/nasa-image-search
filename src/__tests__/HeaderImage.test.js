import React from 'react';
import { render } from '@testing-library/react';
import HeaderImage from '../components/HeaderImage';

test('component HeaderImage renders', () => {
  const { asFragment } = render(<HeaderImage />);
  const component = asFragment();
  expect(component).toMatchSnapshot();
});