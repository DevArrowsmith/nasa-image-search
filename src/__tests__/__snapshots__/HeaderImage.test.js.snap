// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component HeaderImage renders 1`] = `
<DocumentFragment>
  <img
    class="sc-AxjAm ivsrOq"
  />
</DocumentFragment>
`;
